### PR TITLE
Move PDC client to upstream repo

### PR DESCRIPTION
**Pull request by erichuanggit**
JIRA: PDC-949

### CODE REVIEW

File: pdc_client/bin/poc [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+# Changelogs
+# v0.1.5
+# - contact model improvement
+#   * contact_type -> contact_role;
+#   * person_name -> username;
+#   * use rolecontact in components;
+
+version = "0.1.5"
+host_name = "https://pdc.app.qa.eng.nay.redhat.com"  # QA Server
```
Comment (simozhan) [inline pdc_client/bin/poc]: Is it ok to expose this server?

Comment (erichuanggit) [inline pdc_client/bin/poc]: better change to an example url

**Comment by bliuredhat** [comment]
+1

**Comment by simozhan** [comment]
+1

File: pdc_client/bin/pdc_client [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    try:
+        client, session = make_client()
+    except BeanBagException as e:
+        print "%d %s" % (e.response.status_code, e.response.content)
+    except Exception as e:
+        print str(e)
+    else:
+        with RequestMethod(client, session, options.resource,
+                           options.traceback, options.debug) as request:
+            print json.dumps(request.dispatch(options.request, data),
```
Comment (bliuredhat) [inline pdc_client/bin/pdc_client]: The task of PDC-960 will edit this script, I am waiting for this script merged to master branch.

**Comment by lubomir** [comment]
Looks good to me. Please squash before merging.

We should also look if the monkey patch is still needed. It looks like the infinite recursion should be [fixed upstream](https://github.com/requests/requests-kerberos/pull/42). But it is not part of this issue.

**Comment by erichuanggit** [comment]
I will squash them after reviewing by team.

**Comment by erichuanggit** [comment]
@lubomir, I am waiting for the testing result of 0.6.0 scratch build before making the official one. Even already had a new release branch in my local, when with `make rpm`, `get_upstream.sh` will clone latest code from upstream repo. 
Could we stop merging any new change before release out 0.6.0? Otherwise it will involve in new changes which are not in the scope of this release.

**Pull request closed without merging**